frontend/public/components/utils/managed-by.tsx

```diff
@@ -19,38 +17,40 @@ import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager/
 import { ClusterServiceVersionKind } from '@console/operator-lifecycle-manager';
 import { ResourceLink } from '.';
 
-export const ManagedByOperatorResourceLink: React.SFC<ManagerLinkProps> = (props) => {
-  const { csvName, namespace, owner, className } = props;
-  const name = owner.name;
-  const { group, version } = groupVersionFor(owner.apiVersion);
-  const kind = owner.kind;
-
-  const reference = referenceForGroupVersionKind(group)(version)(kind);
-  const { namespaced } = modelFor(reference);
-
-  const link = `/k8s/ns/${namespace}/${referenceForModel(
-    ClusterServiceVersionModel,
-  )}/${csvName}/${reference}/${name}`;
-
+export const ManagedByOperatorResourceLink: React.SFC<ManagerLinkProps> = ({
+  csvName,
+  namespace,
+  owner,
+  className,
+}) => {
+  const csvGroupVersionKind = referenceForModel(ClusterServiceVersionModel);
+  const ownerGroupVersionKind = referenceForOwnerRef(owner);
+  const { apiGroup, kind, namespaced } = modelFor(ownerGroupVersionKind) ?? {};
+  const ownerIsCSV =
+    apiGroup === ClusterServiceVersionModel.apiGroup && kind === ClusterServiceVersionModel.kind;
+  const link = `/k8s/ns/${namespace}/${csvGroupVersionKind}/${csvName}`;
```

Comment:
Not new, but use `resourcePathFromModel` instead of building the URL yourself. This will become important when we start putting the cluster name in URLs.

Reply:
@spadgett  updated